components/occurrences/SecureOccurrenceSection.js

```diff
@@ -38,18 +40,26 @@ const SecureOccurrenceSection = ({ occurrences }) => {
           key={occurrence.name}
           currentOccurrence={occurrence}
           mainText={"Vulnerability Scan"}
-          timestamp={occurrence.completed}
+          timestamp={
+            occurrence.completed
+              ? `Completed at ${dayjs(occurrence.completed).format(
```

Comment:
Maybe a future enhancement is to check how long it's been since the scan started. If it's >24h (or some interval), then we may never have received the final discovery occurrence (e.g., in case of network errors or a bug in a collector).

Reply:
Makes sense to me, this scenario happened to me while testing with the harbor collector so I think adding this safety is a good idea. I'll add it to an issue.